ceph-releases/jewel/ubuntu/14.04/daemon/entrypoint.sh

```diff
@@ -376,7 +380,7 @@ EOF
     chmod +x /etc/service/${CLUSTER}-${OSD_ID}/run
   done
 
-exec /sbin/my_init
+exec /usr/bin/runsvdir -P /etc/service
```

Comment:
Then if we believe if doesn't do anything useful we should remove it, I'll have a closer look to this.

@dmitryint, any comment to this?